Tribler/Core/permid.py

```diff
@@ -47,3 +48,24 @@ def save_pub_key(keypair, pubkeyfilename):
     with open(pubkeyfilename, 'w') as file:
         file.write(membuf.read())
     membuf.close()
+
+
+def generate_keypair_multichain():
+    return LibNaCLSK()
```

Comment:
LibNaCL is always used to generate the tunnel keys. I assume M2Crypto does not have support for the curve used. The `dispersy.crypto.generate_key` function that you suggest only picks M2Crypto or LibNaCL based on a string comparison, so I think it would not be an improvement here. Also note that `LibNaCLSK()` is our own function from dispersy.crypto, not a library function.